app/components/windows/go-live/PlatformSettings.tsx

```diff
@@ -58,7 +58,7 @@ export default class PlatformSettings extends TsxComponent<Props> {
     return (
       <ValidatedForm class="flex" ref="settingsForm">
         <div style={{ width: '100%' }}>
-          {!hasPlatforms && $t('Enable at least one destination to start streaming')}
+          {!hasPlatforms && $t('Enable at least one destinationType to start streaming')}
```

Comment:
i feel like we should not use camelCase in any user-facing copy

Reply:
Looks like another find-replace error

Reply:
My IDE did a bad job with replacing =(